config/nrfconnect/chip-module/Kconfig

```diff
@@ -202,6 +202,52 @@ config CHIP_FACTORY_RESET_ERASE_NVS
 	bool
 	default y if CHIP_FACTORY_DATA || CHIP_FACTORY_DATA_CUSTOM_BACKEND
 
+config CHIP_WIFI_CONNECTION_RECOVERY_MINIMUM_INTERVAL
+	int "Define the minimum connection recovery time interval in milliseconds"
+	depends on CHIP_WIFI
+	default 500
+	help
+	  Specifies the minimum connection recovery interval (in milliseconds).
+
+config CHIP_WIFI_CONNECTION_RECOVERY_MAXIMUM_INTERVAL
+	int "Define the maximum connection recovery time interval in milliseconds"
+	depends on CHIP_WIFI
+	default 3600000 # 1 hour
+	help
+	  Specifies the maximum connection recovery interval (in milliseconds).
+
+config CHIP_WIFI_CONNECTION_RECOVERY_MAX_OVERALL_INTERVAL
+	int "Define maximum amount of connection recovery occurrences"
+	depends on CHIP_WIFI
+	default 0
+	help
+	  If it is a non-zero value specifies the maximum amount of connection 
```

Comment:
I suggest the following description:
*Specifies the maximum connection recovery attempts. If set to 0, no limitation is applied and the connect retry is performed endlessly.*

Reply:
It would be better to implement this as a time limit, in milliseconds. But this refinement can be postponed I guess.